Jenkinsfile

```diff
@@ -79,7 +79,7 @@ pipeline {
                             publishToRepository product: 'slurm',
                                                 format: 'yum',
                                                 maturity: 'stable',
-                                                tech: 'sles12.3',
+                                                tech: 'sles-12',
```

Comment:
Don't we have a pretty well established convention of not using a `-` between the distro name and version?  Is there a particular reason to deviate here?

Reply:
This was done to make the repository names more readable and more easily parsed by scripts maintaining Nexus server, especially when we need to group by distro and then by version.